static/src/js/components/DraftList/DraftList.jsx

```diff
@@ -53,6 +53,38 @@ const DraftList = ({ draftType }) => {
     })
   }
 
+  // Building a Table using Data in items
+  const renderRows = (items.map((item) => {
```

Comment:
Call this `data`, as what you are building is a 2D array, you'll need to move the key into the Table component, but that gets the logic out of the caller.

Reply:
let's huddle on this when you have time. I'm not sure which key you are referring to and I get eslint errors when I try to move keys to table